src/main/java/io/pakland/mdas/githubstats/domain/entity/PullRequest.java

```diff
@@ -54,7 +56,7 @@ public List<Commit> getCommitsByUser(User user) {
       return commits
           .stream()
           .filter(commit -> commit.getUser().equals(user))
-          .toList();
+          .collect(Collectors.toList());
```

Comment:
Why `Collectors` over the `.toList` method?

Reply:
It looks like it's a compatibility change as the `.toList()` version prevented @cjp4 from running locally. Otherwise the methods are equivalent (apart from [these differences](https://stackoverflow.com/questions/65969919/differences-of-java-16s-stream-tolist-and-stream-collectcollectors-tolist) that are not relevant for our case).

Reply:
But what version is he using? We agree to use the v17. Shouldn't we all have the same?